src/rbac/application/project/controller/project_command_controller.py

```
@@ -0,0 +1,41 @@
+    @inject
+    def __init__(
+            self,
+            project_command_usecase: ProjectCommandUseCase = Depends(Provide[Container.project_command_usecase])
+    ):
+        self.project_command_usecase = project_command_usecase
+
+    @router.post("/")
+    async def create_project(
+            self,
+            request: ProjectCreateRequest,
+            user_detail: UserDetail = Depends(get_current_user),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)